crates/nargo_cli/src/cli/check_cmd.rs

```diff
@@ -128,6 +128,7 @@ mod tests {
             typed_param(
                 "d",
                 AbiType::Struct {
+                    name: String::from("mystruct"),
```

Comment:
```suggestion
                    name: String::from("MyStruct"),
```

To satisfy the spellchecker

Reply:
This also need to be applied to the other I instances of "mystruct"

Reply:
Done!

Reply:
Curious why the spellchecker in the CI didn't pick it up though...